docs/refmanual.rst

```diff
@@ -8,4 +8,5 @@ Cascade-AT Reference Manual
    core
    dismod
    executor
+   model
```

Comment:
this almost looks alphabetical, just saying.

Reply:
Well, my mother was a librarian.